### DISCUSSION TITLE
wlroots?

### DISCUSSION THREAD

**Discussion by probonopd** [CuarzoSoftware/Louvre#14] [General]
From what I have learned, using wlroots in a Wayland compositor is  the generally nowadays recommended way to make the compositor compatible with the latest and greatest protocols of the day without having to do all the work.

Is Louvre considering to use it?

**Comment by ehopperdietzel** [comment]
The concept behind Louvre is to maintain a simple API, rather than supporting all protocols. My plan is to include the most standardized and essential protocols for building functional compositors.

While there might be wlroots modules that could be used in Louvre, it depends on the protocol/utility. However, Louvre already has a structure for incorporating protocols. Therefore, I believe that, in most cases, using wlroots might be more complicated than implementing protocols directly and would introduce unnecessary overhead. Of course, anyone can use wlroots to extend Louvre's functionalities if they wish.
